public/js/test/integration/tests.js

```diff
@@ -134,24 +134,24 @@ describe("Tests", function() {
    * * pausing in an uncaught exception
    * * reloading while paused and resuming execution
    */
-  it("(Firefox) exception", function() {
+  it.only("(Firefox) exception", function() {
     debugPage("exceptions.html");
     scopeAtIndex(0).click();
-    scopeAtIndex(1).contains("reachable")
+    scopes().contains("reachable")
 
     resume();
     scopeAtIndex(0).click();
-    scopeAtIndex(1).contains("Error")
+    scopes().contains("Error")
 
     resume();
     scopeAtIndex(0).click();
-    scopeAtIndex(1).contains("Error")
+    scopes().contains("Error")
 
     resume();
     stepOver();
     stepOver();
     scopeAtIndex(0).click();
-    scopeAtIndex(1).contains("unreachable")
+    scopes().contains("unreachable")
```

Comment:
we also need to update the tests to now search any scope for the error because we can't assume it's the first scope property.